README.md

```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
```

Comment:
@sebbert: Yeah, that sounds good. I was afraid of this quickly turning into a `.unoconfig`-tutorial, but your suggestion balances this nicely.